cg/services/delivery_message/messages/rna_delivery_message.py

```diff
@@ -51,31 +52,23 @@ def __init__(self, store, strategy: RNAUploadMessageStrategy):
         self.strategy = strategy
 
     def create_message(self, cases: list[Case]) -> str:
-        if len(cases) == 1:
-            return self._get_case_message(cases[0])
         return self._get_cases_message(cases)
 
-    def _get_case_message(self, case: Case) -> str:
+    def _get_scout_message_for_case(self, case: Case) -> str:
         related_uploaded_dna_cases = self.store.get_uploaded_related_dna_cases(rna_case=case)
+        if not related_uploaded_dna_cases:
+            raise CaseNotFoundError(f"No matching DNA cases for case {case.name}.")
```

Comment:
We can have this in the CRUD layer I reckon. If we try to fetch something that doesn't exist of which we assume it does it should break. so in the get_uploaded_related_dna_cases

Reply:
Changing this!